extensions/WEBGL_shader_pixel_local_storage/extension.xml

```diff
@@ -37,9 +37,20 @@
           <code>FramebufferMemorylessPixelLocalStorageANGLE</code> is not supported.
         </div>
         <div class="note rationale">
-          Browsers must be able to preempt a WebGL context at any moment, and if memoryless pixel
-          local storage planes could be active during such a time, their contents would become
-          invalidated.
+          Browsers must be able to interrupt a WebGL rendering context at any moment, and if
+          memoryless pixel local storage planes could be active during such a time, their contents
+          would become invalidated.
+        </div>
+      </addendum>
+      <addendum>
+        <div class="note">
+          When <code>WEBGL_shader_pixel_local_storage</code> is enabled, the following WebGL
+          extensions are enabled implicitly:
+          <p>
+            <code>OES_draw_buffers_indexed</code><br/>
+            <code>EXT_color_buffer_float</code><br/>
+            <code>EXT_color_buffer_half_float</code>
```

Comment:
Note for the future: WebGL PLS CTS should explicitly check for this (e.g., by trying to create a renderbuffer).

Reply:
Sounds good. Where are the CTS tests? Can I start porting the ANGLE tests to CTS?

Reply:
PLS extension should be in `sdk/tests/conformance2/extensions`, added as a new `.html` file and a new entry in `00_test_list.txt` there.

Reply:
Awesome, thanks!

@kenrussell , if I add tests to `sdk/tests/conformance2/extensions` will they automatically run on Chrome when I add the JS bindings?

Reply:
@csmartdalton yes, if it's structured similarly to the other extension tests, then as soon as a browser implements the extension, the test will start to run. Note the importance of adding it to `00_test_list.txt`.

In Chrome, we'll need to roll forward the WebGL conformance suite in order to pick up the new test on the automated bots.

Reply:
@lexaknyazev is there an easy way to check if an extension is enabled? I can't call `getExtension` since that will automatically enable it. Or do I have to try and use the dependency extensions and make sure they are on?

Reply:
1. Unextended WebGL 2.0 does not support float and half-float renderbuffers. This could be observed by trying to create one and checking for a generated error.
2. The PLS extension implicitly enables float and/or half-float color buffer extensions, therefore float and/or half-float renderbuffer creation may succeed after enabling the PLS extension.

However this seems inconsistent with the ANGLE PLS extension spec:
- Half-float formats are not mentioned at all
- Float format (R32F) is not optional; should the relevant extension be required?

Reply:
You are correct. This is not in the spec yet, but it's one of the issues in the current working doc:

https://docs.google.com/document/d/1BsXxar1gbJ_w-Xvy76VbgmLOgB3qON6BY6YB5LVOqwo/edit

The spec needs to be updated to mark float formats as dependent on EXT_color_buffer_float